src/tasks/publish/index.ts

```diff
@@ -117,6 +117,9 @@ async function publishTask(
   const selectedNetwork = bre.network.name
   const ipfsApiUrl =
     ipfsApiUrlArg || (bre.config.ipfs || {}).ipfsApi || defaultIpfsApiUrl
+  const verifyContract =
+    (bre.config.etherscan && Boolean(bre.config.etherscan.apiKey)) || verify
```

Comment:
If we don't have an Etherscan API key, can we still verify the contract?

Reply:
Good point. I decided to have an extra early sanity check in case the user wants to verify and don't have an API Key setup.